force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -122,13 +163,11 @@ public with sharing class FlowControlLoopsCollections {
      * @return The sum of all integers up to the limit.
      */
     public static Integer sumUpToLimit(Integer intLimit) {
-        /* 
-         for (){
-            
-         }
-        */
-
-        return null; // Replace null with the variable you used to store the result
+        Integer sum = 0;
+        for (Integer i = 1; i <= intLimit; i++) {
+            sum += i;
+        }
+        return sum;
```

Comment:
Good work, and good job initializing your index variable to 1. A lot of students miss that.